### PR TITLE
[embed] embed v3 api and demo notebook update

### PR DESCRIPTION
**Pull request by alekhya-n**

### CODE REVIEW

**Review by andrewvonrosenbach — DISMISSED** [review]
I can't comment on the code but lgtm.

Also noting, we need this notebook to be published today, just to validate the final URL for the notebook is correctly linked from the product listing.

After that, I'll submit another PR with the updated ARNs for all the files.

Could we also update the naming convention of the v2 notebook to make it explicit that it refers to the v2 listing and model? Can be in a separate PR if that's simpler.

File: notebooks/Deploy embed v3 model.ipynb [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO: replace with the cohere embed v3 model packages when ready\n",
+    "cohere_package = \"cohere-english-small-v2-2-5c5864d6b4ee3277bbbca80225b5638f\"\n",
```
Comment (jpekmez) [inline notebooks/Deploy embed v3 model.ipynb:87]: don't we want to use the same notebook for all v3 packages?

**Review by jpekmez — DISMISSED** [review]

**Comment by jpekmez** [comment]
the input_field is already in this [PR](https://github.com/cohere-ai/cohere-aws/pull/77)

File: notebooks/Deploy embed v3 model.ipynb (hunk 1 above)
Comment (lfayoux) [reply notebooks/Deploy embed v3 model.ipynb:87]: I thought we had it separate for english and multilingual

**Review by lfayoux — APPROVED** [review]

File: notebooks/Deploy embed v3 model.ipynb (hunk 1 above)
Comment (jpekmez) [reply notebooks/Deploy embed v3 model.ipynb:87]: today we don't have english embed on sagemaker

**Pull request merged**